Add seamonkey to the top of the list of gtkmozembed widgets to try. This fixes the broken seamonkey support reported in #215149.
--- extconf.rb.~1~	2006-12-29 14:17:28.000000000 +0100
+++ extconf.rb	2008-03-29 08:28:39.166594383 +0100
@@ -3,7 +3,8 @@
 =end
 
 PACKAGE_NAME = "gtkmozembed"
-PACKAGE_IDS = ["xulrunner-gtkmozembed", 
+PACKAGE_IDS = ["seamonkey-gtkmozembed",
+               "xulrunner-gtkmozembed", 
                "firefox-gtkmozembed", 
                "mozilla-gtkmozembed"]
 
